ts/sbd-server/src/common.ts

```diff
@@ -0,0 +1,108 @@
+import { fromUint8Array, toUint8Array } from 'js-base64';
```

Comment:
The contents are this file are just consolidated from other files, no code changes here.